CHANGELOG.md

```diff
@@ -1,5 +1,62 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.10.17
+
+<!-- generated comparing v5.10.16..master -->
+
+_Dec 5, 2022_
+
+A big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- @mnajdova added a Material You Button playground (#35222)
```

Comment:
Not sure if it is worth to be highlighted, but if we decide to do it, let’s add a link to it :)